oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/trace/UninstrumentedGatewaysConfig.java

```diff
@@ -62,13 +61,15 @@ private void activeSetting(String config) {
             log.debug("Updating using new static config: {}", config);
         }
         this.settingsString.set(config);
-        onGatewaysUpdated(parseGatewaysFromYml(config));
+        if (config != null) {
+            onGatewaysUpdated(parseGatewaysFromYml(""));
+        }
```

Comment:
You might mean `if (config == null)` update with `""`? The same in other places in this PR

```suggestion
        if (config == null) {
            onGatewaysUpdated(parseGatewaysFromYml(""));
        }
```

Reply:
yes, mistake that cause ci failed